src/components/TargetDetails/parseUtils.tsx

```diff
@@ -118,13 +118,27 @@ export const parseContent = (text: string | undefined) => {
   });
 };
 
-export const parseExamples = (example: string) => {
-  const lines = splitByNewLine(example);
+export const parseExamples = (example: string | string[]) => {
+  let lines: string[] | undefined = [];
+  if (typeof example === 'object') {
+    example.forEach(e => {
+      const splits = splitByNewLine(e);
+      if (splits) {
+        splits.forEach(s => {
+          if (lines) {
+            lines.push(s);
```

Comment:
Instead of iterating over each element of `splits` and adding them individually, it's more performant to just concatenate `splits` to `lines`. You can do so like this:
```js
const splits = splitByNewLine(e);
if (splits && lines) lines.concat(splits)
```